signer/local_cosigner_test.go

```diff
@@ -104,39 +76,11 @@ func TestLocalCosignerSign2of2(t *testing.T) {
 	signState2, err := LoadOrCreateSignState(stateFile2.Name())
 	require.NoError(t, err)
 
-	signerTypeConfig1 := LocalSoftSignThresholdEd25519SignatureConfig{
-		CosignerKey: key1,
-		RsaKey:      *rsaKey1,
-		Total:       total,
-		Threshold:   threshold,
-	}
-	cosignerConfig1 := LocalCosignerConfig{
-		SignState: &signState1,
-		Peers:     peers,
-	}
-
-	signerTypeConfig2 := LocalSoftSignThresholdEd25519SignatureConfig{
-		CosignerKey: key2,
-		RsaKey:      *rsaKey2,
-		Total:       total,
-		Threshold:   threshold,
-	}
-	cosignerConfig2 := LocalCosignerConfig{
-		SignState: &signState2,
-		Peers:     peers,
-	}
-
-	var cosigner1 Cosigner
-	var cosigner2 Cosigner
-
-	localsigner1 := NewLocalSigner(SignerType, signerTypeConfig1)
-	cosigner1 = NewLocalCosigner(cosignerConfig1, localsigner1)
-
-	localsigner2 := NewLocalSigner(SignerType, signerTypeConfig2)
-	cosigner2 = NewLocalCosigner(cosignerConfig2, localsigner2)
+	localSigner1 := NewLocalSoftSignThresholdEd25519Signature(key1, threshold, total)
+	cosigner1 := NewLocalCosigner(1, "", peers, &signState1, localSigner1)
 
-	require.Equal(t, cosigner1.GetID(), 1)
-	require.Equal(t, cosigner2.GetID(), 2)
```

Comment:
I know its dumb but maybe let it be or add that all ids are the same?

Reply:
this is covered by `TestLocalCosignerGetID` so we don't need to make these assertions here